esp/services/ws_fs/ws_fsService.cpp

```diff
@@ -105,6 +105,28 @@ void CFileSprayEx::init(IPropertyTree *cfg, const char *process, const char *ser
     
     xpath.clear().appendf("Software/EspProcess[@name=\"%s\"]/EspService[@name=\"%s\"]/QueueLabel", process, service);
     cfg->getProp(xpath.str(), m_QueueLabel);
+    StringArray qlist;
+    getDFUServerQueueNames(qlist, nullptr);
+    if (qlist.ordinality())
+    {
+        if (!m_QueueLabel.length())
+            m_QueueLabel.append(qlist.item(0));
+        else
+        {
+            bool found = false;
```

Comment:
I think that queue name is case-insensitive. The contains() may not work if m_QueueLabel does not match the queue name case stored inside the qlist. Or, I have to force the queue names in the qlist to be lowercase. Not sure it is worth.